### ISSUE TITLE
Tab dragging is broken (also affects tearing off tabs and moving tabs between windows)

### ISSUE THREAD

**Issue by mattatobin** [MoonchildProductions/Tycho#60]
Tab dragging is rather busted likely due to e10s non-sense.

Tabs cannot be dragged around on the tab bar and can't be torn off to create a new window. Moving tabs from one window to another does not move them and the target window gains an inclosable empty tab.

**Comment by wolfbeast** [comment]
Anything in the error console?

**Comment by trav90** [comment]
```
Timestamp: 06/08/2016 08:30:17 PM
Error: TypeError: invalid 'in' operand draggedTab._dragData
Source File: chrome://browser/content/tabbrowser.xml
Line: 3565
```
```
Timestamp: 06/08/2016 08:30:17 PM
Error: NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIXPCComponents_Utils.isCrossProcessWrapper]
Source File: resource://gre/modules/PageThumbUtils.jsm
Line: 71
```

**Comment by wolfbeast** [comment]
CPOWs. Yup, that's e10s. :/ I knew these were going to bite me sometime doing this change.
The first error should be simple enough to fix. Probably some old-age JS usage that needs to be updated for syntax.

**Comment by wolfbeast** [comment]
A quick look indicates the main problem isn't the dragging itself, but rather creating the thumbnail to show hovered with the mouse cursor. Definitely e10s/async and the related refactoring of browsers and DOM content windows at fault here. Possibly the quickest workaround is to pull in our previous PageThumbs.jsm, its related Utils module, and the worker script to /browser; it's a utility at most. Otherwise, disabling this hovered thumbnail is an option until we find a better solution.

**Comment by wolfbeast** [comment]
Tab dragging fixed in ad08ec9e4817974bb328cf79f67c7a8910528104 
Tab tear-off fixed in 6693c1fbde497c5763745336d87f1f5ecb34b01e
